src/library/sidebarmodel.cpp

```diff
@@ -257,8 +257,13 @@ QVariant SidebarModel::data(const QModelIndex& index, int role) const {
             // If it's the "Quick Links" node, display it's name
             if (pTreeItem->getData().toString() == QUICK_LINK_NODE) {
                 return pTreeItem->getLabel();
+            } else {
+                if (pTreeItem->getToolTip().isEmpty()) {
+                    return pTreeItem->getData();
+                } else {
+                    return pTreeItem->getToolTip();
+                }
```

Comment:
Let's hide the useless Ids, except when debugging
```suggestion
            } else {
                if (!pTreeItem->getToolTip().isEmpty()) {
                    return pTreeItem->getToolTip();
                } else if (CmdlineArgs::Instance().getDeveloper()) {
                    return pTreeItem->getData();
                }
```

Reply:
775e7c7ff86cc3811238426901e1082043a2f6b9